src/Tribe/Admin_List.php

```diff
@@ -155,6 +158,41 @@ public static function sort_by_event_date( Array $clauses, WP_Query $wp_query )
 			return $clauses;
 		}
 
+		/**
+		 * Defines custom logic for filtering events table by aggregator record.
+		 *
+		 * @param   Array       $clauses    SQL clauses for fetching posts
+		 * @param   WP_Query    $wp_query   A paginated query for items
+		 *
+		 * @return  Array                   Modified SQL clauses
+		 */
+		public static function filter_by_aggregator_record( Array $clauses, WP_Query $wp_query ) {
+			// Check for event post type.
+			if ( $wp_query->get( 'post_type' ) !== Tribe__Events__Main::POSTTYPE ) {
+				return $clauses;
+			}
+
+			// Check if filtering by aggregator record.
+            $record_num = intval( tribe_get_request_var( 'aggregator_record', 0 ) );
+            if( $record_num === 0 ) {
```

Comment:
* :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after IF keyword; 0 found
* :no_entry_sign: There must be a space between if and the first parenthesis '('
* :no_entry_sign: Use Yoda Condition checks, you must.

<sub>hash: 09998dbbab378d9bc48a04799cb2f737</sub>